README.md

```diff
@@ -998,6 +1004,22 @@ to `true` in your playbooks using the `ha_cluster` role.
     - linux-system-roles.ha_cluster
 ```
 
+### Creating pcsd certificate and private key files using the `certificate` role
+
+This example creates self-signed pcsd certificate and private key files
+in /var/lib/pcsd with the file name FILENAME.crt and FILENAME.key, respectively.
+
+```yaml
+- hosts: node1 node2
+  vars:
+    ha_cluster_pcsd_certificates:
+      - name: FILENAME
```

Comment:
Pcsd only looks for `pcsd.crt` and `pcsd.key` files. The `name` variable is even set by the `ha_cluster` role:
```
{%   set _ = _cert.__setitem__('name', '/var/lib/pcsd/pcsd') %}
```
Therefore, I see no point in letting customers set the `name` variable. I think that the readme should state, that the variable is set automatically and should not be changed.